### PR TITLE
Add video carousel section to code.org/music

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds the video carousel section to https://code.org/music ahead of the launch next week.

## Links
Jira ticket: [ACQ-1840](https://codedotorg.atlassian.net/browse/ACQ-1840)

## Testing story
Tested locally

----

https://github.com/code-dot-org/code-dot-org/assets/9256643/c4cab7a9-6c56-4c5c-b160-2e089fb98f73

## Backup videos

https://github.com/code-dot-org/code-dot-org/assets/9256643/d6135ce2-57bd-416d-881c-4d793a6bd4de

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -2013,6 +2013,7 @@
     music_lab: "Music Lab"
     introducing_music_lab: "Introducing Music Lab"
     introducing_music_lab_strong: "Introducing **%{music_lab}**"
+    make_music_learn_code: "Make Music, Learn Code"
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: This won't be translated in time for launch.

File: pegasus/sites.v3/code.org/views/music/music_lab_carousel_videos.haml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    {
+      code: "Fysf1rD-RX0",
+      download_path: "//videos.code.org/music_lab/Music_Lab_Rosa_Linn.mp4",
+      caption: hoc_s("music_lab.artist.singer_songwriter", markdown: :inline, locals: {artist_name: "Rosa Linn"}),
+    },
+    {
+      code: "1TDma4wg3dE",
+      download_path: "//videos.code.org/music_lab/Music_Lab_Drew_Taggart_from_The_Chainsmokers.mp4",
+      caption: "Drew Taggart from The Chainsmokers",
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/views/music/music_lab_carousel_videos.haml]: should this be translated?

**Review by bethanyaconnor — APPROVED** [review]

File: pegasus/sites.v3/code.org/views/music/music_lab_carousel_videos.haml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/views/music/music_lab_carousel_videos.haml]: Oh, yes! I forgot we already had a string for this 🙌 Updating now.

Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/views/music/music_lab_carousel_videos.haml]: Here's a screenshot with the updated string working:
Screenshot 2024-05-15 at 12 25 13 PM

**Pull request merged**